homeassistant/components/zha/helpers.py

```diff
@@ -1025,9 +1025,9 @@ def cluster_command_schema_to_vol_schema(schema: CommandSchema) -> vol.Schema:
     """Convert a cluster command schema to a voluptuous schema."""
     return vol.Schema(
         {
-            vol.Optional(field.name)
-            if field.optional
-            else vol.Required(field.name): schema_type_to_vol(field.type)
+            (
```

Comment:
Same as before, cosmetic and shouldn't be in this PR.

Reply:
This has been automatically done by the ruff formatter.